README.md

```diff
@@ -41,21 +41,29 @@ Unlike `vde_vmnet`, `socket_vmnet` does not depend on VDE.
 Requires macOS 10.15 or later.
 
 ### From binary
-
+ Get the latest release version - 
 ```bash
 VERSION="$(curl -fsSL https://api.github.com/repos/lima-vm/socket_vmnet/releases/latest | jq -r .tag_name)"
 FILE="socket_vmnet-${VERSION:1}-$(uname -m).tar.gz"
+```
 
-# Download the binary archive
+Download the binary archive
+```bash
 curl -OSL "https://github.com/lima-vm/socket_vmnet/releases/download/${VERSION}/${FILE}"
+```
 
-# (Optional) Attest the GitHub Artifact Attestation using GitHub's gh command (https://cli.github.com)
+ (Optional) Attest the GitHub Artifact Attestation using GitHub's gh command (https://cli.github.com)
+ ```bash
 gh attestation verify --owner=lima-vm "${FILE}"
+```
```

Comment:
The problem with this part is that most people will skip this part since this is optional. I think we want to help people to start using attestations.

Also the text is not helpful because most user do not know that is "GitHub Artifact Attestation". We should replace this with a short sentence using term that most users already know, and link to relevant docs adding more info.

We can use an alert like this:

> [!TIP]
> To verify the downloaded binary using GitHub's gh command (https://cli.github.com), run:
> ```bash
> gh attestation verify --owner=lima-vm "${FILE}"
> ```
> Learn more about [GitHub Artifact Attestation](https://docs.github.com/en/actions/security-for-github-actions/using-artifact-attestations/using-artifact-attestations-to-establish-provenance-for-builds)